### PR TITLE
create nss passwd file from existing passwd file

### PR DESCRIPTION
**Pull request by bparees**
fixes https://github.com/openshift/postgresql/issues/61

### CODE REVIEW

**Comment by bparees** [comment]
@mfojtik ptal

File: 9.2/root/usr/share/container-scripts/postgresql/common.sh [hunk 1]
```diff
@@ -90,7 +90,8 @@ function generate_postgresql_recovery_config() {
 function generate_passwd_file() {
   export USER_ID=$(id -u)
   export GROUP_ID=$(id -g)
-  envsubst < "${CONTAINER_SCRIPTS_PATH}/passwd.template" > "${HOME}/passwd"
+  cat /etc/passwd | grep -v ^postgres > "$HOME/passwd"
```
Comment (mfojtik) [inline 9.2/root/usr/share/container-scripts/postgresql/common.sh]: `grep -v ^postgres /etc/passwd > ...`

**Comment by mfojtik** [comment]
one small nit, LGTM

**Comment by bparees** [comment]
de-nitted.

**Pull request merged**